packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -503,18 +519,28 @@ export class ExecutionEngineHttp implements IExecutionEngine {
     // handle nethermind buggy response
     // see: https://discord.com/channels/595666850260713488/1293605631785304088/1298956894274060301
     if (
+      method === "engine_getBlobsV1" &&
```

Comment:
might have been fixed, confirm and remove this snippet of handling buggy resp

Reply:
Confirmed, fixed in Nethermind [v1.30.0](https://github.com/NethermindEth/nethermind/releases/tag/1.30.0). I removed it